### PR TITLE
Use zh-TW for language code, instead of zh_Hant.

### PR DESCRIPTION
**Pull request by sonata-chen**
This pull request fixed issue [#667](https://github.com/LemmyNet/lemmy-ui/issues/667).

### CODE REVIEW

**Comment by sonata-chen** [comment]
This pull request fix issue [#667](https://github.com/LemmyNet/lemmy-ui/issues/667).

File: src/shared/i18next.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const languages = [
   { resource: sv, code: "sv", name: "Svenska" },
   { resource: vi, code: "vi", name: "Tiếng Việt" },
   { resource: zh, code: "zh", name: "中文 (简体)" },
-  { resource: zh_Hant, code: "zh_Hant", name: "中文 (繁體)" },
+  { resource: zh_Hant, code: "zh-TW", name: "中文 (繁體)" },
```
Comment (dessalines) [inline src/shared/i18next.ts:66]: Must use an underscore.

**Review by dessalines — CHANGES_REQUESTED** [review]

File: src/shared/i18next.ts (hunk 1 above)
Comment (sonata-chen) [reply src/shared/i18next.ts:66]: Could you elaborate more why we must use an underscore here?
I have tested on my machine. It seems that it's not OK to use underscores for both zh_TW and  pt_BR.
Lemmy (without a login) will not pick the correct language based on browser settings.

Comment (sonata-chen) [reply src/shared/i18next.ts:66]: My browser setting
`navigator.languages:   [ "zh-TW", "en-US", "en" ]`

Comment (dessalines) [reply src/shared/i18next.ts:66]: Ah... I was under the impression it needed one, coulda swore I tested that with pt-BR. In that case I'll trust your judgement here, thx!

**Review by dessalines — APPROVED** [review]

**Pull request merged**